### PR TITLE
about us and footer

### PR DESCRIPTION
**Pull request by DoctorHeisenberg**

### CODE REVIEW

**Review by SidletskyiOlexandr — APPROVED** [review]

File: src/app/presentation/components/home-components/about/about.component.html [hunk 1]
```diff
@@ -31,9 +31,9 @@ <h1 class="registration_title">About Us</h1>
...
-                      <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore.</p>
-                      <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>
+                      <p>We are the young team, we are from different cities, we have different education, but we are united by one goal – to become the great developers.</p>
+                      <p>We have known each other in all for three month, but despite this we are choosing, planning, and starting the development of our project. Many of using technologies and frameworks was new for us, and we spend lot of time on the beginning to investigate it. And additionally nobody worked with SCRUM before. But we have done a great job, and you will see that from first minutes in our application</p>
```
Comment (papich3pich) [inline src/app/presentation/components/home-components/about/about.component.html]: Replace "in all" with "only", "month" with "months"
Many of **_used_** technologies and frameworks was new for us, and we spen**_t_**  **_a_** lot of time **_at_** the beginning to investigate it.
**_Also_** nobody **_has_** worked with SCRUM before. **_However_**, we have done great job. You'll see **_it_** from first minutes in our application.
Split this paragraph into few lines, HTML allows that.

File: src/app/presentation/components/home-components/about/about.component.html [hunk 2]
```diff
@@ -31,9 +31,9 @@ <h1 class="registration_title">About Us</h1>
...
+                      <p>We using many technologies, such as: Angular, C#, Entity Framework, SQL, ASP.NET, GIT, Azure. Most of them was new for us, but we done it wery well. </p>
```
Comment (papich3pich) [inline src/app/presentation/components/home-components/about/about.component.html]: Many technologies were used, such as: Angular, C#, Entity Framework, SQL, ASP.NET, GIT, Azure. We have done this project, despite the fact that it was new to us.

File: src/app/presentation/components/home-components/about/about.component.html [hunk 3]
```diff
@@ -46,9 +46,9 @@ <h3 class="feature-title">About team</h3>
...
               <div class="about-text text-right justify-start">
                 <h3 class="feature-title">About our product</h3>
-                <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>
-                <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore.</p>
-                <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>
+                <p>In my country there is a problem with small buisnes, because of problems with managment. Company owners doesnt want to spending money for managers, justifying this by the fact that their company doesnt so big, and all employers can manage their job on their own. And as a rule, thats strategy leads to the chaos, and total misunderstanding between owners and employees.</p>
```
Comment (papich3pich) [inline src/app/presentation/components/home-components/about/about.component.html]: In my country there is a problem with small **_business_**, **_caused by_** problems with managment. Company owners don't want to spend money for managers, explaining that their company isn't so big, and all employees can manage their job on their own. Obviously, this strategy leads to chaos and total misunderstanding among owners and employees.

File: src/app/presentation/components/home-components/about/about.component.html [hunk 4]
```diff
@@ -46,9 +46,9 @@ <h3 class="feature-title">About team</h3>
...
+                <p>Our goal is to have all work live in Provis, while simplify the application as much as possible, with retaining all the functionality - thereby making people more productive and giving back at least 15% of time to dedicate to other things. </p>
```
Comment (papich3pich) [inline src/app/presentation/components/home-components/about/about.component.html]: Our goal is to gather all tasks in Provis, while simplifying the application as much as possible with retaining all the functionality - thereby making people more productive and giving back at least 15% of time to dedicate to other things.

File: src/app/presentation/components/home-components/about/about.component.html [hunk 5]
```diff
@@ -46,9 +46,9 @@ <h3 class="feature-title">About team</h3>
...
+                <p>With our system you can comfortable: delegating, commenting tasks, follow up on assignments, keep track of deadlines, observe the efficiency of employees, better understand how much time a specific task takes, and based on this knowledge better plan your business in the future. And this is only part of the possibilities that Provis can propose you.</p>
```
Comment (papich3pich) [inline src/app/presentation/components/home-components/about/about.component.html]: With our system you can comfortably: delegate, comment tasks, follow up on assignments, keep track of deadlines.
Also it allows you to observe the efficiency of employees, better understand how much time a specific task takes. Based on this knowledge you will plan your business better in the future. This is the least that Provis can propose you.

File: src/app/presentation/components/home-components/about/about.component.html [hunk 6]
```diff
@@ -63,10 +63,10 @@ <h3 class="feature-title">About our product</h3>
     <section class="about-top-section bottom-section d-flex">
         <div class="about-section">
           <div class="titled">
-              <span>Some title</span>
+              <span>Let's start</span>
             </div>
             <div class="bottom-section-text">
-              <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>
+              <p>We hope that you are ready for start the new era in yours buisnes. Because after using Provis, you will can't return to your past way of managing. Good luck!</p>
```
Comment (papich3pich) [inline src/app/presentation/components/home-components/about/about.component.html]: We hope that you are ready to start the new era of your business. After using Provis, you won't be able to return to your previous way of managing. Good luck!

**Review by papich3pich — APPROVED** [review]

**Pull request merged**